tests/Doctrine/Tests/DBAL/Platforms/AbstractPostgreSqlPlatformTestCase.php

```diff
@@ -976,4 +976,15 @@ public function testQuotesDatabaseNameInCloseActiveDatabaseConnectionsSQL()
             true
         );
     }
+
+    public function testGetDefaultValueDeclarationSQLEscaped() : void
+    {
+        // string must be escaped
+        $field = [
+            'type' => 'string',
+            'default' => 'Foo\\Bar'
+        ];
+
+        self::assertSame(" DEFAULT 'Foo\\Bar'", $this->_platform->getDefaultValueDeclarationSQL($field));
```

Comment:
Unlike the functional tests, this one covers one special character out of 13.

Reply:
Fixed by removing.